.github/scripts/check_small_models.py

```diff
@@ -0,0 +1,66 @@
+"""
+Check all models marked as size "small" in the manifest with TLC. Small
+models should finish executing in less than ten seconds on the GitHub CI
```

Comment:
How reliable (flaky tests) is this assumption given unpredictable load on shared CI infrastructure?

Reply:
Pretty reliable. It finishes executing in about 2.5 minutes.